CMakeLists.txt

```diff
@@ -292,8 +292,8 @@ source_group( "" FILES ${SOURCE_GROUP_TOPLEVEL})
 # ToDo: make configurable
 
 install(FILES ${PROJECT_SOURCE_DIR}/src/kvazaar.pc DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
+install(TARGETS kvazaar-bin DESTINATION ${CMAKE_INSTALL_BINDIR})
 install(TARGETS kvazaar
-  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
```

Comment:
Why leave this out? `RUNTIME DESTINATION` covers the .dll on Windows.

Reply:
Ouch, misunderstood.